### PR TITLE
Add environment path tho example file

### PR DESCRIPTION
**Pull request by SvenNederhoff**
Without this addition Errbot cant find the `git` command which makes it impossible to use `!repo install`

### CODE REVIEW

File: docs/code_examples/systemd.service [hunk 1]
```diff
@@ -4,6 +4,7 @@ After=network.service
 
 [Service]
 Environment="LC_ALL=en_US.UTF-8"
+Environment="PATH=/usr/bin:/path/to/errbot/virtualenv/bin"
```
Comment (zoni) [inline docs/code_examples/systemd.service]: We should probably set `/bin/` if we're setting `$PATH` here.

Comment (SvenNederhoff) [reply docs/code_examples/systemd.service]: maybe add the full default path ? like `/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin`

Comment (zoni) [reply docs/code_examples/systemd.service]: The term "default" is a bit subjective and OS-vendor dependent, but that seems sensible yes.

**Comment by SvenNederhoff** [comment]
>The term "default" is a bit subjective and OS-vendor dependent, but that seems sensible yes.

the term Common use would be more aplicible

**Pull request merged**